multisrc/overrides/peachscan/default/additional.gradle

```diff
@@ -0,0 +1,5 @@
+dependencies {
+    // Only PeachScan sources uses the image-decoder dependency.
```

Comment:
```suggestion
```

Reply:
I think the comment should be left in, I usually prefer explanations for why a lint/inspection had to be disabled. In this case, it's because only PeachScan and no one else uses this dependency, so adding it to the Gradle version catalog isn't worth it.